src/sagemaker/config/__init__.py

```diff
@@ -93,6 +93,8 @@
     AUTO_ML_VOLUME_KMS_KEY_ID_PATH,
     AUTO_ML_INTER_CONTAINER_ENCRYPTION_PATH,
     ENDPOINT_CONFIG_DATA_CAPTURE_KMS_KEY_ID_PATH,
+    SESSION_S3_BUCKET_PATH,
```

Comment:
Can we quickly discuss about this naming, want to make sure we are using session correctly.

Reply:
SESSION_DEFAULT_S3_BUCKET_PATH
SESSION_DEFAULT_S3_OBJECT_KEY_PREFIX_PATH